backend/src/main/java/heartbeat/util/IdUtil.java

```diff
@@ -26,4 +22,8 @@ static String getSourceControlReportId(String timeStamp) {
 		return SOURCE_CONTROL_PREFIX + timeStamp;
 	}
 
+	static String getDataCompletedPrefix(String timeStamp) {
+		return DATA_COMPLETED_PREFIX + timeStamp;
```

Comment:
timeStamp is not readable

Reply:
At present, the time stamp transmitted by the front-end is more accurate to find the corresponding generated file based on the time stamp